src/main/java/org/jitsi/jigasi/transcription/VoskTranscriptionService.java

```diff
@@ -213,20 +219,26 @@ public void onMessage(String msg)
                 partial = false;
                 result = obj.getString("text");
             }
+
             if (!result.isEmpty() && !result.equals(lastResult))
```

Comment:
Can you also check if the result is not partial here to ensure that the final result is always sent even if it's the same as the previous interim result?
e.g. if (!result.isEmpty() && (!partial || !result.equals(lastResult)))

Reply:
Oooo good point, I missed that part. Thank you @TomDurrant-yt